include/swift/SIL/AbstractionPattern.h

```diff
@@ -497,6 +559,18 @@ class AbstractionPattern {
     return pattern;
   }
 
+  /// Return an abstraction pattern for the partially-applied curried
+  /// type of an C++ method.
```

Comment:
Could you repeat the example here?  The partially-applied curried type will be `(Compartment, Temperature) -> ()`.